DependencyInjection/SonataAdminExtension.php

```diff
@@ -203,6 +203,8 @@ public function load(array $configs, ContainerBuilder $container)
         // set filter persistence
         $container->setParameter('sonata.admin.configuration.filters.persist', $config['persist_filters']);
 
+        $container->setParameter('sonata.admin.configuration.mosaic.button.show', $config['show_mosaic_button']);
```

Comment:
shpuld be `...show.mosaic.button` instead of `...mosaic.button.show`

poing @greg0ire